src/prefs.ts

```diff
@@ -117,114 +133,123 @@ function settings_dialog_view(): [AppWidgets, Gtk.Container] {
         margin_end: 10,
         margin_bottom: 10,
         margin_top: 10,
-    })
+    });
 
     const win_label = new Gtk.Label({
         label: "Show Window Titles",
         xalign: 0.0,
-        hexpand: true
-    })
+        hexpand: true,
+    });
 
     const snap_label = new Gtk.Label({
         label: "Snap to Grid (Floating Mode)",
-        xalign: 0.0
-    })
+        xalign: 0.0,
+    });
 
     const smart_label = new Gtk.Label({
         label: "Smart Gaps",
-        xalign: 0.0
-    })
+        xalign: 0.0,
+    });
+
+    const unstack_label = new Gtk.Label({
+        label: "Automatically destroy single-window stacks",
+        xalign: 0.0,
+    });
 
     const show_skip_taskbar_label = new Gtk.Label({
         label: "Show Minimize to Tray Windows",
-        xalign: 0.0
-    })
+        xalign: 0.0,
+    });
```

Comment:
It's probably best if you go ahead and revert the unnecessary formatting changes.